src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -413,7 +413,13 @@ private string GenerateRow(string[] values, ReadOnlySpan<int> alignment, Display
                         sb.Append(StringUtil.Padding(_startColumn));
                     }
                 }
+
                 sb.Append(GenerateRowField(values[k], _si.columnInfo[k].width, alignment[k], dc, addPadding));
+                if (values[k].Contains("\u001b"))
```

Comment:
We could use char const (more informative) and `IndexOf(ReserChar)` (more fast).